common/src/model_registration/register.py

```diff
@@ -181,40 +202,28 @@ def main(args):
 
     # validations
     if model_type not in SUPPORTED_MODEL_ASSET_TYPES:
-        tc.track_event(name="FM_import_pipeline_debug_logs",
-                       properties={"message": f"Unsupported model type {model_type}"})
-        tc.flush()
+        tc_log(f"Unsupported model type {model_type}")
         raise Exception(f"Unsupported model type {model_type}")
 
     if not model_name:
-        tc.track_event(name="FM_import_pipeline_debug_logs",
-                       properties={"message": "Missing Model Name. Provide model_name as input \
-                                   or in the model_download_metadata JSON"})
-        tc.flush()
+        tc_log("Missing Model Name. Provide model_name as input or in the model_download_metadata JSON")
         raise Exception("Missing Model Name. Provide model_name as input or in the model_download_metadata JSON")
 
     # check if we can have lineage and update the model path for ws import
     if not registry_name and args.model_import_job_path:
-        tc.track_event(name="FM_import_pipeline_debug_logs",
-                       properties={"message": "Using model output of previous job as \
-                                   run lineage to register the model"})
-        tc.flush()
+        tc_log("Using model output of previous job as run lineage to register the model")
         print("Using model output of previous job as run lineage to register the model")
         with open(args.model_import_job_path) as f:
             model_import_job_path = json.load(f)
         model_path = model_import_job_path.get("path", model_path)
     elif model_type == AssetTypes.MLFLOW_MODEL:
         if not os.path.exists(os.path.join(model_path, MLFLOW_MODEL_FOLDER)):
-            tc.track_event(name="FM_import_pipeline_debug_logs",
-                           properties={"message": f"Making sure, model parent directory is `{MLFLOW_MODEL_FOLDER}`."})
-            tc.flush()
+            tc_log(f"Making sure, model parent directory is `{MLFLOW_MODEL_FOLDER}`")
             print(f"Making sure, model parent directory is `{MLFLOW_MODEL_FOLDER}`.")
```

Comment:
If you're going to follow every tc_log statement with a print with the same info, consider adding a generic print statement in tc_log.

Reply:
Makes Sense